hack/olm-registry/olm-artifacts-template.yaml

```diff
@@ -0,0 +1,447 @@
+            properties:
+              apiVersion:
+                description: 'APIVersion defines the versioned schema of this representation
+                  of an object. Servers should convert recognized schemas to the latest
+                  internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
+                type: string
+              kind:
+                description: 'Kind is a string value representing the REST resource
+                  this object represents. Servers may infer this from the endpoint
+                  the client submits requests to. Cannot be updated. In CamelCase.
+                  More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+                type: string
+              metadata:
+                type: object
+              spec:
+                description: RouteMonitorSpec defines the desired state of RouteMonitor
+                properties:
+                  route:
+                    description: Route is the resource that holds the name and Namespace
+                      of the Route to monitor
+                    properties:
+                      name:
+                        description: Name is the name of the Route
+                        type: string
+                      namespace:
+                        description: Namespace is the namespace of the Route
+                        type: string
+                    type: object
+                type: object
+              status:
+                description: RouteMonitorStatus defines the observed state of RouteMonitor
+                properties:
+                  routeURL:
+                    description: RouteURL is the url extracted from the Route resource
+                    type: string
+                type: object
+            type: object
+        version: v1alpha1
+        versions:
+        - name: v1alpha1
+          served: true
+          storage: true
+      status:
+        acceptedNames:
+          kind: ''
+          plural: ''
+        conditions: []
+        storedVersions: []
+    - apiVersion: v1
+      kind: Service
+      metadata:
+        creationTimestamp: null
+        labels:
+          control-plane: controller-manager
+        name: route-monitor-operator-cmms
+      spec:
+        ports:
+        - name: https
+          port: 8443
+          targetPort: https
+        selector:
+          control-plane: controller-manager
+      status:
+        loadBalancer: {}
+    - apiVersion: rbac.authorization.k8s.io/v1beta1
+      kind: ClusterRole
+      metadata:
+        creationTimestamp: null
+        name: route-monitor-operator-metrics-reader
+      rules:
+      - nonResourceURLs:
+        - /metrics
+        verbs:
+        - get
+    - apiVersion: operators.coreos.com/v1alpha1
+      kind: ClusterServiceVersion
```

Comment:
The CSV should be in the generated bundle, all the should exist in this for operator install is the subscription, catalogsource and operatorgroup

Reply:
If this is the case, where would you point me to look for a good example?
Also I took the information from cloud-ingress-operator..
Is that repo doing the generation incorrectly aswell?

Reply:
If this is the case, where would you point me to look for a good example?
Also I took the information from.